### DISCUSSION TITLE
'Smart Fill' paint-on supports

### DISCUSSION THREAD

**Discussion by maltyjava** [supermerill/SuperSlicer#2763] [General]
This is pretty much the only feature I miss from Prusa Slicer, would it be possible to add into SS?

Both the 'smart fill' and the 'triangle' brush option are really useful, I find.

**Comment by supermerill** [comment]
they aren't available in the 2.4 beta?

**Comment by maltyjava** [comment]
Oh nice, I was still on Gravel. Just downloaded the latest and they're there! Thanks
